From 6305dd094c6c6cfa9b3897cc26ac5b10c8433b06 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Thu, 7 Sep 2017 12:28:50 +0800
Subject: [PATCH 2565/5242] MLK-16375-2 Revert "MLK-16349 video: fbdev: dcss:
 reset dcss domains before clock source config"

commit  f94ff062c5d0124d72ef5adb23a1ec58e9fe2f58 from
https://source.codeaurora.org/external/imx/linux-imx.git

The reverted patch is to workaround a hang issue which is caused
by DCSS reset control bug in 'Block Control' submodule. But this
workaround may cause another issue on some boards. So revert it.

This reverts commit 2d22e932664dc0347dc279a5588834a29aec47ce.

(cherry picked from commit 6f1ae20eb4a12af5179d859890fa32d5042a17cd)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxc/imx_dcss.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/video/fbdev/mxc/imx_dcss.c b/drivers/video/fbdev/mxc/imx_dcss.c
index 72695e6..790be11 100644
--- a/drivers/video/fbdev/mxc/imx_dcss.c
+++ b/drivers/video/fbdev/mxc/imx_dcss.c
@@ -3254,8 +3254,7 @@ static int dcss_probe(struct platform_device *pdev)
 	if (ret)
 		goto kfree_info;
 
-	/* reset DCSS to make sure in reset state*/
-	writel(0xffffffff, info->blkctl_base + 0x8);
+	/* TODO: reset DCSS to make it clean */
 
 	/* Clocks select: before dcss de-resets */
 	if (!strcmp(info->disp_dev, "hdmi_disp"))
-- 
1.7.9.5

